### DISCUSSION TITLE
How to add A extension?

### DISCUSSION THREAD

**Discussion by unknown07724** [TurboWarp/extensions#1570] [General]
how to add it?
Am I stupid?

**Comment by samuellouf** [comment]
Have a look [there](https://github.com/TurboWarp/extensions/blob/master/CONTRIBUTING.md) !
Maybe it will not be added for different reaons :

- The moderators and admins are busy/in vacation/sick
- They didn't have the time to check your commit
- Your extensions has problems (ex: Geolocation extension -> You get ppl's location -> That's creepy/dangerous)

Hope I helped you 😄 !

**Comment by snowboyz0825** [comment]
Do you want to add an extension that is able to be downloaded by everyone or just an extension to turbowarp as a whole

**Comment by unknown07724** [comment]
imma close this, I learned how to
